test/e2e-test/nextjs.test.ts

```diff
@@ -72,14 +72,7 @@ describe('Create nextjs application', () => {
     it('init template nextjs"', async () => {
         console.log('Creating Next project');
         const expectOutput = `Export successful. Files written to ${template}/out`
-        await execFile(`npx -y create-next-app@latest ${template}`);
-        const data = fs.readFileSync(`${template}/pages/index.js`);
-        const dataToString = data.toString();
-        const dataToArray = dataToString.split('\n');
-        dataToArray.splice(1,1); //Remove import of NextJS/Image
-        dataToArray.splice(61,1,'               <img src="/vercel.svg" alt="Vercel Logo" width="72" height="16" />\n');
-        const arrayToText =  dataToArray.join('\n');
-        fs.writeFileSync(`${template}/pages/index.js`,arrayToText)
+        await execFile(`npx -y create-next-app@latest --example basic-css ${template}`);
         process.chdir(template);
         await execFile(`npx next build`);
         const { stdout } = await execFile(`npx next export`);
```

Comment:
Can't comment on the next line, so will have to do it over here.
Mixed feelings about the assertion/this test. I mean, the fix expects that we don't have NextJS/Image import, and neither .svg loads, but the assertion doesn't check for that. Can we assert for that explicitly?

Reply:
This is a quick fix for the crashing test case.
There are other more elaborate possibilities, such as using a more elaborate regex to handle certain tags in the html file, or creating your own template in a repository so you don't depend on something external.